digitalpy/core/zmanager/impl/zmq_pusher.py

```diff
@@ -0,0 +1,44 @@
+import zmq
+from digitalpy.core.zmanager.pusher import Pusher
+from digitalpy.core.zmanager.request import Request
+from digitalpy.core.parsing.formatter import Formatter
+
+class ZMQPusher(Pusher):
+    
+    def __init__(self, formatter: Formatter):
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D107 Missing docstring in __init__